src/main/java/avd/inf/jdm/rentmycar/service/BookingService.java

```diff
@@ -45,6 +42,15 @@ public Booking save(Booking booking) {
         return bookingRepository.save(booking);
     }
 
+    public Booking startRide(Booking booking) {
```

Comment:
This new method is exactly what I meant! We create a new `Ride` on the `Booking`, `set` it and update the `BookingStatus `👍 

I am only confused why we are returning a `Booking`. Do you need that somewhere? Otherwise we can either make it a `void`, or (my preference) we return a `boolean` that lets the calling actor know if the `startRide` method has succeeded. What do you think?

And should `startRide` check if the `booking` exists? Or do you think the method will not be able to be called with an invalid `booking`?